shuup/admin/modules/orders/static_src/create/view/lines.js

```diff
@@ -145,13 +180,40 @@ export function orderLinesView(store, isCreating) {
         ),
         m("div.list-group", renderOrderLines(store, shop.selected, lines)),
         m("hr"),
-        m("button.btn.text-success" + (isCreating ? ".disabled": ""), {
-            disabled: isCreating,
-            onclick: () => {
-                if(!isCreating) {
-                    store.dispatch(addLine());
-                }
-            }
-        }, m("i.fa.fa-plus"), " " + gettext("Add new line"))
+        m("div.row", [
+            m("div.col-sm-6",
+                m("button.btn.btn-success" + (isCreating ? ".disabled": ""), {
+                    disabled: isCreating,
+                    onclick: () => {
+                        store.dispatch(addLine());
+                        var productId = productQuickSelect.currentProduct();
+                        if (productId){
+                            store.dispatch(retrieveProductData(
+                                {id: productId, forLine: _.last(store.getState().lines).id}
+                            ));
+                        }
```

Comment:
Also add some clear selection icon. Since the once selected option seems to be pretty hard to clear.